packages/base-styles/_variables.scss

```diff
@@ -91,14 +113,24 @@ $spinner-size: 16px;
 $canvas-padding: $grid-unit-20;
 
 /**
- * Editor widths.
+ * Mobile specific styles
+ */
+$mobile-text-min-font-size: 16px; // Any font size below 16px will cause Mobile Safari to "zoom in".
+
+/**
+ * Editor styles.
  */
 
 $sidebar-width: 280px;
 $content-width: 840px;
 $wide-content-width: 1100px;
 $widget-area-width: 700px;
 $secondary-sidebar-width: 350px;
+$editor-font-size: 16px;
+$default-block-margin: 28px; // This value provides a consistent, contiguous spacing between blocks.
+$text-editor-font-size: 15px;
```

Comment:
Should this be `$font-size-large`, or do we want to keep it hardcoded to make sure it won't be affected by changes to the type scale?

Reply:
To be honest I'm not sure about the full purpose of this variable, so didn't want to change it here :) 

It seems to relate to the mobile app?

Reply:
🤷  not really sure, but I'm ok with keeping it unchanged